gcloud/cloudbuild.yaml

```diff
@@ -4,15 +4,21 @@
 steps:
 - name: 'gcr.io/cloud-builders/docker'
```

Comment:
Instead of an Arg, would it make more sense to have two Dockerfiles in the source? Dockerfile.slim builds gcloud-slim, and Dockerfile.full starts with `FROM gcloud-slim` and adds components. WDYT?

Reply:
Say that's smart!

Unfortunately, in order to be `FROM gcr.io/cloud-builders/gcloud-slim` you have to first publish an image there. So for now this change builds two separate unrelated images, and once we've built `gcloud-slim` once we can change `gcloud` to build `FROM` it. WDYT?

Reply:
Nevermind, as soon as I wrote that I realized that was dumb and could do it a smarter way. PTAL!